src/components/challenge/Challenge.js

```diff
@@ -1,11 +1,103 @@
 import React, { Component } from 'react';
 
+const OverviewSection= () => {
+  return (
+      <div className="container-fluid">
+        <div className="row">
+          <h2 className="challenge-section-title">Habit Overview</h2>
+          <p className="challenge-section-desc">
+            Over the past few weeks, you've been learning a lot about what
+            you can do to improve your well-being. This week is an opportunity
+            for you to do some research on your own! The focus of this week
+            is on how foods can affect how we feel, both physically and
+            emotionally. Besides helping you stay in shape, healthy foods can
+            help you increase your energy, boost your immune system,
+            minimize inflammation, promote muscle building, and more.
+            <a href="/challenge"> Learn more about the science behind healthy eating</a>.
+          </p>
+        </div>
+      </div>
+  )
+}
+
+const ChallengeSection = () => {
+  return (
+      <div className="container-fluid">
+        <div className="row">
+          <h2 className="challenge-section-title">Challenge</h2>
+          <p className="challenge-section-desc">Lorem Ipsum</p>
+        </div>
+      </div>
+  )
+}
+
+const PrizesSection= () => {
+  return (
+      <div className="container-fluid">
+        <div className="row">
+          <h2 className="challenge-section-title">Prizes</h2>
+          <p className="challenge-section-desc">Lorem Ipsum</p>
+        </div>
+      </div>
+  )
+}
+
+const SuggestionsSection = () => {
+  return (
+    <div className="container-fluid">
+      <div className="row">
+        <h2 className="challenge-section-title">Suggestions</h2>
+        <p className="challenge-section-desc">
+          <b>Need a few ideas or suggestions to help get you started?</b>
```

Comment:
I usually try to avoid using `<b>` tags in general and instead try to handle it all in the css. This makes it easier to modify how it looks in the future from only one location, especially if we want to say, "hey instead of bold, I think italics looks good here" or "instead of bold, let's just change the color".